client/web/src/search/input/SearchPage.tsx

```diff
@@ -101,7 +101,7 @@ export const SearchPage: React.FunctionComponent<SearchPageProps> = props => {
     return (
         <div className="web-content search-page d-flex flex-column align-items-center pb-5">
             <BrandLogo className="search-page__logo" isLightTheme={props.isLightTheme} variant="logo" />
-            {props.isSourcegraphDotCom && <div className="search-page__cloud-tag-line">Search public code</div>}
+            {!props.isSourcegraphDotCom && <div className="text-muted mt-3">Search public code</div>}
```

Comment:
Was the inversion of the condition intentional? We shouldn't say search "public" code on private instances, right?
```suggestion
            {props.isSourcegraphDotCom && <div className="text-muted mt-3">Search public code</div>}
```

Reply:
whooooooooops, the inversion was for testing, thanks for catching